File: grid.py
class OutputGrid:

  columnmap = {
    "0": "A",
    "1": "B",
    "2": "C",
    "3": "D",
    "4": "E"
  }
  column = 0
  row = 0
  while column < 5:
    row = 0
    while row < 5:
      dictkey = columnmap[str(column)] + str(row+1)
      print(grid[dictkey], end=" ")
      row += 1
    print("")
    column += 1

File: app/src/main/java/Games/grid/Grid.java
package Games.grid;

import Games.ship.Ship;

public class Grid {
    private final int GRIDSIZE = 5;
    private char[][] grid = new char[GRIDSIZE][GRIDSIZE];

    public Grid(){
        for (int column = 0; column < GRIDSIZE; column++){
            for (int row = 0; row < GRIDSIZE; row++){
                grid[column][row] = 'o';
            }
        }
    }

    public void printGrid(){
        System.out.println("  ABCDE");
        System.out.println("  -----");
        for(int column = 0; column < GRIDSIZE; column++){
            System.out.print(column+1);
            System.out.print("|");
            for(int row = 0; row < GRIDSIZE; row++){
                System.out.print(grid[column][row]);
            }
            System.out.println("");
        }
    }

    public void placeShip(Ship ship){

    }

}
